### PR TITLE
#293 add stategeocadastre-ukr-cyrl-latn-1993

### PR DESCRIPTION
**Pull request by CAMOBAP**
- #293

### CODE REVIEW

File: maps/stategeocadastre-ukr-Cyrl-Latn-1993.yaml [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+    "\u041d": "N"        # Н
+    "\u041e": "O"        # О
+    "\u041f": "P"        # П
+    "\u0420": "R"        # Р
+    "\u0421": "S"        # С
+    "\u0422": "T"        # Т
+    "\u0423": "U"        # У
+    "\u0424": "F"        # Ф
+    "\u0425": "Kh"       # Х
+    "\u0426": "Ts"       # Ц misstype in original doc TZ
```
Comment (ronaldtse) [inline maps/stategeocadastre-ukr-Cyrl-Latn-1993.yaml]: @CAMOBAP How do we know it is a typo that Ts/Tz?

Comment (CAMOBAP) [reply maps/stategeocadastre-ukr-Cyrl-Latn-1993.yaml]: Well, it's a good question, I have not 100% evidence.

1. I tried to search (google) values with `TZ` but wasn't able to find anything. But with `ts`
2. According to pronunciation `ts` closer than `tz`

Comment (CAMOBAP) [reply maps/stategeocadastre-ukr-Cyrl-Latn-1993.yaml]: 3. Also, any other transliteration system didn't use `tz` but use `ts`

Comment (ronaldtse) [reply maps/stategeocadastre-ukr-Cyrl-Latn-1993.yaml]: Sounds good, can you document it in notes and we can merge this? Thanks!

**Review by ronaldtse — APPROVED** [review]

File: maps/stategeocadastre-ukr-Cyrl-Latn-1993.yaml (hunk 1 above)
Comment (CAMOBAP) [reply maps/stategeocadastre-ukr-Cyrl-Latn-1993.yaml]: Done, once we will get official response I may update this

**Pull request merged**